webpack.common.config.js

```diff
@@ -25,6 +25,7 @@ var entry = {
     // JS
     'base-page': staticPath('js/pages/base-page.js'),
     'home-page': staticPath('js/pages/home-page.js'),
+    'landing-page': staticPath('js/pages/landing-page.js'),
```

Comment:
At this point I'd rather see `home-page.js` still be what shows up at `/`, with a different URL location and JavaScript filename for the quicksearch—though it seems to me that the search functionality on this PR makes more sense as a feature on the new dashboard than as the sole feature on a standalone page. But perhaps @pattisdr @caneruguz you can say more about the direction we're headed? I gather there's a whole Epic [happening](https://openscience.atlassian.net/browse/OSF-5374). ;-)